docs/_docs/commands/szurubooru.js

```diff
@@ -48,8 +48,15 @@ async function setTagCategory(name, category) {
 (async () => {
     // Szurubooru doesn't use the same ratings as most boorus so we need to map them
     const ratingsMap = {
+        "": "safe",
```

Comment:
Not sure about this one, if there's no rating shouldn't we leave it to Szurubooru to decide what to use instead of  being opinionated there?

Reply:
It's not about being opinionated, rather that this script is coded in such a way that lacking a proper rating makes images not go through. The original script as it is will simply not work for images rated:sensitive, which skips a lot of images for absolutely no reason whatsoever. Inherently this is meant for localboorus, i.e. people who want to have their own local archive for one reason or another. Pointlessly sticking to semantics achieves nothing but create headaches for users who will eventually notice this and will have to wrangle to fix this.

Reply:
I have no idea what happens if the image has no rating at all, because I don't know an easy example to refer to and use with, but it's a catch-all that should catch it provided it comes through, rather than just skipping it.